test/cookbooks/test/recipes/php.rb

```diff
@@ -0,0 +1,39 @@
+::Chef::Recipe.include Apache2::Cookbook::Helpers
+
+directory 'purge distro conf.modules.d' do
+  extend Apache2::Cookbook::Helpers
+  path "#{apache_dir}/conf.modules.d"
+  recursive true
+  action :nothing # trigger this in other resources as needed
+end
+
+directory 'purge distro conf.d' do
+  extend Apache2::Cookbook::Helpers
+  path "#{apache_dir}/conf.d"
+  recursive true
+  action :nothing # trigger this in other resources as needed
+end
```

Comment:
It would be really preferred to have this included in a more global fashion as it can be used in other places in this cookbook. This is troublesome since we've been only doing it in a recipe however we'd need to figure out a way to do this via a library method I believe.

@Xorima @tas50 any ideas on how we can do that in a sane way?

Reply:
So I discussed this with @tas50 today and we decided it's best if we just include this resource/logic directly in the ``apache2_mod_php`` resource so that it will work "out of the box". For other resources we'll just have to replicate this code. Since it's running inside of a resource context, we don't need to worry about duplicated resources.